group_vars/all.yml

```diff
@@ -22,5 +22,6 @@ grafana_security:
   admin_user: admin
   admin_password: password
 
-
+probe_url:
```

Comment:
can we rename this to something more expressive like `prometheus_http_probe_urls`

Reply:
renamed.